prez-components/src/components/PrezUINode.vue

```diff
@@ -1,34 +1,90 @@
 <script lang="ts" setup>
-import type { PrezUINodeProps } from '../types';
+import { RouterLink } from "vue-router";
 import Chip from "primevue/chip";
+import { PrezUINodeProps } from "../types";
 
 const props = defineProps<PrezUINodeProps>();
-
-const label = (props.term.label ? props.term.label.text :
-    (props.term.curie ? props.term.curie : props.term.uri));
-
-let tooltip = (props.term.description ? props.term.description.text : (props.term.curie ? props.term.curie : props.term.uri));
-if(tooltip == label) {
-    // we don't want to show a tooltip with the same info as the label
-    if(label != props.term.uri) {
-        // if the label doesn't show the uri, then show the uri as the tooltip
-        tooltip = props.term.uri;
-    } else {
-        // we have nonthing useful to show
-        tooltip = undefined;
-    }
-}
 </script>
+
 <template>
-    <a v-if="props.size == 'small'" :href="props.term.uri">
-        <Chip v-tooltip.bottom="tooltip" :label="label" />
-    </a>
-    <a v-else :href="props.term.uri" v-tooltip="tooltip">
-        {{ label }}
-    </a>
+    <div class="node">
+        <component
```

Comment:
Should we keep router decisions in the main Prez-UI app, so it can manage them, any other options like firing events? How would this work with the application using these prez components, how do the components stay independent?  What happens if it's rendered as a custom element?

Reply:
I think the app using the custom element should be able to handle the link as needed. But, happy to discuss after the PR.